wp/wp-content/themes/lahma_maker/template-parts/content-arcsi.php

```diff
@@ -93,7 +128,12 @@ function sortAssociativeArrayByKey($array, $key, $direction){
 
 <?php 
 // if Episode is archived
-if ($showarchived) { ?>
+if ($showarchived) { 
+    
+$fullTitle = $showtitle . ' | ' . $showname;
+    
+?>
+
 
 <div class="arcsiblokk">
```

Comment:
Shouldn't this part be conditional of the outcome of `isDomainAvailible`?

Reply:
yes, it is, else is not needed explicitly. 
if(isDomainAvailible) check is not needed if passes the negation

Reply:
Why? The code will still be executed (and it will hang if arcsi is down). I would expect an else branch on line 93.

Reply:
Everythin executes only if L97 statement evaluates true which means domain has to be returning `true` too if I'm not mistaken. Right? I think this would work then

Reply:
Yes, after the additonal check in https://github.com/mmmnmnm/lahmacun/pull/388/commits/93a7b45d00099256e40e207ace78a1f4fdba5727 was added